nextflow_schema.json

```diff
@@ -1,47 +1,520 @@
+                },
+                "observations_type": {
+                    "type": "string",
+                    "default": "sample",
+                    "description": "Type of observation we have- often 'sample'"
+                }
+            },
+            "required": ["observations_id_col", "observations_type"]
+        },
+        "features_options": {
+            "title": "Features options",
+            "type": "object",
+            "description": "Options related to features",
+            "default": "",
+            "help_text": "",
+            "properties": {
+                "features_id_col": {
+                    "type": "string",
+                    "default": "gene_id",
+                    "description": "Feature ID attribute in the GTF file (e.g. the gene_id field)"
+                },
+                "features_name_col": {
+                    "type": "string",
+                    "default": "gene_name",
+                    "description": "Feature name attribute in the GTF file (e.g. the gene symbol field)"
+                },
+                "features_type": {
+                    "type": "string",
+                    "default": "gene",
+                    "description": "Type of feature we have, often 'gene'"
+                }
+            },
+            "required": ["features_id_col", "features_name_col", "features_type"]
+        },
+        "filtering": {
+            "title": "Filtering",
+            "type": "object",
+            "description": "Options related to filtering upstream of differential analysis",
+            "default": "",
+            "properties": {
+                "filtering_min_abundance": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "Minimum abundance value"
+                },
+                "filtering_min_samples": {
+                    "type": "number",
+                    "default": 1,
+                    "description": "Minimum observations that must pass the threshold to retain the row/ feature (e.g. gene)."
+                },
+                "filtering_min_proportion": {
+                    "type": "number",
+                    "description": "A minimum proportion of observations that must pass the threshold. Overrides  minimum_samples"
+                },
+                "filtering_grouping_var": {
+                    "type": "string",
+                    "help_text": "The variable can be used to define groups and derive a minimum group size upon which to base minimum observation numbers. The rationale for this is to allow retention of features that might be present in only one group. Note that this is consciously NOT filtering with an explicit awareness of groups (\"feature must be present in all samples of group A\"), since this is know to create biases towards discovery of differential features.",
```

Comment:
```suggestion
                    "help_text": "The variable can be used to define groups and derive a minimum group size upon which to base minimum observation numbers. The rationale for this is to allow retention of features that might be present in only one group. Note that this is consciously NOT filtering with an explicit awareness of groups (\"feature must be present in all samples of group A\"), since this is known to create biases towards discovery of differential features.",
```